Apps/ClinicalDocument/src/Models/ClinicalDocumentRecord.cs

```diff
@@ -26,13 +26,13 @@ public class ClinicalDocumentRecord
         /// <summary>
         /// Gets or sets the ID.
         /// </summary>
-        [JsonPropertyName("id")]
+        [JsonPropertyName("healthDataId")]
         public string Id { get; set; } = string.Empty;
 
         /// <summary>
         /// Gets or sets the file ID.
         /// </summary>
-        [JsonPropertyName("fileId")]
+        [JsonPropertyName("healthDataFileId")]
```

Comment:
This is the model that gets sent to our front-end.  I thought we should have a separate model for the data returned by PHSA and use a mapper to convert them in the service.  Can discuss after stand-up.